openeo-geotrellis/src/main/scala/org/openeo/geotrellis/OpenEOProcesses.scala

```diff
@@ -708,14 +719,13 @@ class OpenEOProcesses extends Serializable {
         SpacePartitioner[K](kb)
       }
     } else {
+      // At least one partitioner is undefined.
       logger.info(s"Merging cubes with partitioners: ${leftCube.partitioner} - ${rightCube.partitioner} - many band case detected: $manyBands")
-      if(manyBands) {
-        val index: PartitionerIndex[K] = getManyBandsIndexGeneric[K]()
-        SpacePartitioner[K](kb)(implicitly,implicitly,index)
-      }else{
-        SpacePartitioner[K](kb)
-      }
-
+      val nrBands = leftCount.getOrElse(1) + rightCount.getOrElse(1)
+      val outputCellType = maybeCellType(leftCube).getOrElse(DoubleCellType).union(maybeCellType(rightCube).getOrElse(DoubleCellType))
+      val tileSize = maybeTileSize(leftCube).getOrElse(128)
```

Comment:
this is wrong: geotrellis tilesize is actually returning width*height, so something needs to change

Reply:
checking out usage of tilesize in getPartitionerIndexForMaxPartitionSize, the default should perhaps be 128*128
some scaladoc might also help to make sure that interpretation is clear